src/GraphQL/Types/GraphTypesLookup.cs

```diff
@@ -195,10 +195,9 @@ public IEnumerable<IGraphType> All()
                 }
 
                 IGraphType type;
-                var name = typeName.TrimGraphQLTypes();
                 lock (_lock)
```

Comment:
@sungam3r I haven't run my test yet, but I already see you removed it from the right place. In ideal world lock next to it should be removed as well because its a disaster for any possible async code.

Reply:
> In ideal world lock next to it should be removed as well because its a disaster for any possible async code.

If it's not on the hot path with high contention, then there is no problem.